src/components/SinglePastCommitInfo.tsx

```diff
@@ -141,15 +139,30 @@ export class SinglePastCommitInfo extends React.Component<
         <div className={commitStyle}>
           <div className={commitOverviewNumbers}>
             <span>
-              <span className={classes(iconStyle, numberofChangedFilesStyle)} />
+              {defaultIconRegistry.iconReact({
+                name: 'file',
+                className: classes(iconStyle),
+                tag: 'span',
+                title: '# Files Changed'
+              })}
               {this.state.filesChanged}
             </span>
             <span>
-              <span className={classes(iconStyle, insertionIconStyle)} />
+              {defaultIconRegistry.iconReact({
+                name: 'git-insertionsMade',
+                className: classes(iconStyle),
+                tag: 'span',
+                title: '# Insertions'
+              })}
               {this.state.insertionCount}
             </span>
             <span>
-              <span className={classes(iconStyle, deletionIconStyle)} />
+              {defaultIconRegistry.iconReact({
+                name: 'git-deletionsMade',
+                className: classes(iconStyle),
+                tag: 'span',
+                title: '# Deletions'
+              })}
```

Comment:
Use `DefaultIconReact` instead of `defaultIconRegistry.iconReact`, don't need the call to the `classes` function, and should prefer the use of standard JSX syntax:

```jsx
              <DefaultIconReact
                name={'git-deletionsMade'}
                className={iconStyle}
                tag={'span'}
                title={'# Deletions'}
              />
```

The same applies to the other two icons that are getting loaded similarly above.